packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/demos/Chatbot.md

```diff
@@ -85,7 +87,7 @@ This demo displays a basic chatbot, which includes:
 
 This demo displays an embedded chatbot. Embedded chatbots are meant to be placed within a page in your product. This demo includes:
 
-1. A [PatternFly page](/components/page) with a sidebar and masthead
+1. A [PatternFly page](/components/page) with a sidebar, skip to chatbot link, and masthead
```

Comment:
```suggestion
1. A [PatternFly page](/components/page) with a sidebar, "back to bottom" button, and masthead.
```
as long as we can say "button"

Reply:
It doesn't always take you to the bottom per se - it takes you to the chatbot toggle or chatbot content. It's meant to make life a little easier for folks using screenreaders since they can skip navigation, etc. and go right to the chatbot. It also provides them with context that there will be a chatbot.

Reply:
Okay updated this suggestion now that I know what I should be looking at!